lib/stack_master/commands/apply.rb

```diff
@@ -64,7 +64,8 @@ def create_stack
           failed!("Stack creation aborted")
         end
         upload_files
-        cf.create_stack(stack_options.merge(tags: proposed_stack.aws_tags))
+        on_failure = @config.stack_defaults['on_failure'] || 'ROLLBACK'
```

Comment:
AFAIK this only works for CREATE_STACK, so it makes sense to apply it to all stacks. Once a stack has been created successfully, it can be updated after a rollback.